app/views/govuk_publishing_components/components/_share_links.html.erb

```diff
@@ -52,7 +52,7 @@
               'track-action': link[:icon],
               'track-options': track_options
             },
-            class: "govuk-link gem-c-share-links__link #{brand_helper.color_class}" do %>
+            class: "govuk-link govuk-link--no-underline gem-c-share-links__link #{brand_helper.color_class}" do %>
```

Comment:
Somewhat out of scope of this, however I would expect these links to have a base-state of underlined (they are links).  

As a Design change maybe worthy of an issue?

![Jun-01-2021 13-38-33](https://user-images.githubusercontent.com/71266765/120324622-bcbd8f80-c2de-11eb-8454-e7d1cf4efc4b.gif)

Reply:
I'm aware that a few govuk designers are investigating this at the moment. Could be included in future work on making links consistent?